### DISCUSSION TITLE
Supported Hardware

### DISCUSSION THREAD

**Discussion by aoweis** [srsran/srsRAN_Project#47] [General Help] [answered]
Hi,

Can I use any USRP model on the gNB side? Are B200 and B210 supported?

Many thanks,

Ahmad

**Comment by andrepuschmann** [comment, marked as answer]
Yes, both are supported. But just to be clear and before anyone asks, the N200 is not supported. The USRP needs to support NR sample rates.

**Reply by saxywarrior** [reply]
Is there any plans to add support for other SDRs such as a bladeRF?

**Reply by ankuramistry** [reply]
Is N300 supported?

**Reply by andrepuschmann** [reply]
N300 is supported through UHD.
